src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -284,17 +284,45 @@ private async Task<List<ReleaseInfo>> ParseUserSearchAsync(TorznabQuery query)
                         // ignore sub groups info row, it's just an row with an info about the next section, something like "Dual Áudio" or "Legendado"
                         if (row.QuerySelector(".edition_info") != null)
                             continue;
-                        var qDetailsLink = row.QuerySelector("a[href^=\"torrents.php?id=\"]");
+
+                        var torrentInfoEl = row.QuerySelector("div.torrent_info");
+                        var seasonEl = row.QuerySelector("a[href^=\"torrents.php?torrentid=\"]");
```

Comment:
Move vars `torrentInfoEl`, `seasonEl` close to the first usage. That makes the code more readable.

Reply:
Done